src/FSharpSource.targets

```diff
@@ -181,6 +181,9 @@
     <DefineConstants>$(DefineConstants);TODO_REWORK_SERVER</DefineConstants>
     <DefineConstants>$(DefineConstants);NO_LOADER_OPTIMIZATION</DefineConstants>
     <DefineConstants>$(DefineConstants);SILVERLIGHT_COMPILER_FSHARP_CORE</DefineConstants>
+    <DefineConstants>$(DefineConstants);NETSTANDARD1_5</DefineConstants>
```

Comment:
@dsyme ,  we weren't going to add features based on platform except for the existing ones net40 net45 etc, are you sure you want to bake this into the compiler source?